bin/moodle-docker-compose

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -e
 
+if [ -z "$MOODLE_DOCKER_WWWROOT" ];
```

Comment:
IMHO it should be better to check it via `! -d`.

HTH,
Matteo

Reply:
Good catch. I've updated the PR.